### PR TITLE
Exclude .DS_Store in XCFramework slice glob

### PR DESCRIPTION
**Pull request by mattrobmattrob**
Let me know if there's a more wholistic Bazel way to reference these types of excluded macOS files. `.DS_Store` files were causing different action keys across machines.

### CODE REVIEW

**Review by segiddins — COMMENTED** [review]
Nice! Mind adding the exclude for the non-xcframework framework globs as well?

**Comment by mattrobmattrob** [comment]
> Nice! Mind adding the exclude for the non-xcframework framework globs as well?

Let me know if this isn't what you were referencing, @segiddins.

**Review by segiddins — APPROVED** [review]

**Comment by mattrobmattrob** [comment]
Do you mind merging, @segiddins? I don't have write access to the repo yet. Let me know if there's some pattern for how they get merged/approved otherwise!

**Comment by segiddins** [comment]
I can't merge until the builds all pass on CI

**Pull request merged**
